playbooks/linode-headers.yml

```diff
@@ -0,0 +1,23 @@
+#
+# Linode's grub2 image doesn't include kernel headers, which means the
```

Comment:
"grub2 image" This doesn't make any sense. When you specify Grub2 to Linode, it's just specifying the code the CPU jumps to on startup, not some particular "OS image"